### PR TITLE
Bump dotnet tracer version to 1.30.1

### PR DESCRIPTION
**Pull request by pierotibou**
.NET Release: https://github.com/DataDog/dd-trace-dotnet/releases/tag/v1.30.1

Updates the release version to 1.9.2
Updates the development version to 0.1.53-prerelease

### CODE REVIEW

**Review by andrewlock — APPROVED** [review]

File: dotnet/build-packages.sh [hunk 1]
```diff
@@ -1,8 +1,8 @@
 
-RELEASE_VERSION="1.9.1"
-DEVELOPMENT_VERSION="0.1.52-prerelease"
+RELEASE_VERSION="1.9.2"
```
Comment (zacharycmontoya) [inline dotnet/build-packages.sh:2]: We already have a version 1.9.2 that was just posted: https://www.nuget.org/packages/Datadog.AzureAppServices.DotNet/ . Was that the same deliverable as this?

Comment (pierotibou) [reply dotnet/build-packages.sh:2]: Yes I've uploaded it as I thought there shouldn't be any perf issues. 
I'm following this doc: https://datadoghq.atlassian.net/wiki/spaces/APMINT/pages/2263679666/Azure+App+Services+Site+Extension+.NET+Tracer#Release-process
Where it states that we merge after the upload

Comment (zacharycmontoya) [reply dotnet/build-packages.sh:2]: Oh, I think we usually upload the DevelopmentVerification.DdDotNet.Apm dev package while the PR is active, and then once the PR is merged we upload the Datadog.AzureAppServices.DotNet official package

**Review by colin-higgins — APPROVED** [review]

File: dotnet/build-packages.sh (hunk 1 above)
Comment (colin-higgins) [reply dotnet/build-packages.sh:2]: Yes, usually we merge and then get the nuget package from master

**Pull request merged**